scripts/helm/helm_handle_chart.sh

```diff
@@ -33,7 +33,7 @@ fi
 bash $SCRIPTS_DIR/deploy/before-deploy.sh "$HELM_CHART_DIRECTORY"
 
 # set kube config
-if [[ "${KUBE_CONFIG_PATH}" == "" ]] || [[ "${KUBE_CONFIG_PATH}" == "''" ]] || [[ "${KUBE_CONFIG_PATH}" == "None" ]]; then
+if [[ ! -f $KUBE_CONFIG_FILE ]] && ([[ "${KUBE_CONFIG_PATH}" == "" ]] || [[ "${KUBE_CONFIG_PATH}" == "''" ]] || [[ "${KUBE_CONFIG_PATH}" == "None" ]]); then
```

Comment:
What's the different between `KUBE_CONFIG_FILE` and `KUBE_CONFIG_PATH`? If we're trying to have to variables for the same thing why don't we just put:
```
KUBE_CONFIG_PATH=$KUBE_CONFIG_FILE
```
At the top of this block?

Reply:
one's set by [helm's schema](https://github.com/bitovi/bitops/blob/master/scripts/helm/bitops.schema.yaml#L52), and the other was the original internal stuff.  we could probably re-work that a bit to handle it earlier on, but i don't think it should block this PR